packages/components/code-input/index.js

```diff
@@ -76,19 +76,32 @@ const CodeInput = (props) => {
   const elements = [];
   for (let i = 0; i < characters; i++) {
     if (i === 3) elements.push(<hr key="InputCode-line" />);
-
-    elements.push(
-      <input
-        key={`InputCode-${i}`}
-        onChange={handleOnChange}
-        onKeyDown={handleOnKeyDown}
-        onFocus={handleOnFocus}
-        onPaste={handleOnPaste}
-        ref={(el) => (inputsRef.current[i] = el)}
-        maxLength={1}
-        disabled={isDisabled}
-      />
-    );
+    if (interfaceDirection === "rtl")
+      elements.unshift(
+        <input
+          key={`InputCode-${i}`}
+          onChange={handleOnChange}
+          onKeyDown={handleOnKeyDown}
+          onFocus={handleOnFocus}
+          onPaste={handleOnPaste}
+          ref={(el) => (inputsRef.current[i] = el)}
+          maxLength={1}
+          disabled={isDisabled}
+        />
+      );
+    else
+      elements.push(
+        <input
```

Comment:
Move it to a common element and simplify the "if else" condition.

Reply:
deleted the conditions